src/custom/hooks/useRefetchPriceCallback.ts

```diff
@@ -114,14 +114,28 @@ function _handleQuoteError({
       }
       default: {
         // some other operator error occurred, log it
+        // and set price/fee to undefined
         console.error(error)
-        // Clear the quote
-        return clearQuote({ chainId: quoteData.chainId, token: quoteData.sellToken })
+        return setQuoteError({
+          ...quoteData,
+          fee: undefined,
+          price: undefined,
+          lastCheck: Date.now(),
+          error: QuoteErrorCodes.UNHANDLED_ERROR
+        })
       }
     }
   } else {
     // non-operator error log it
+    // and set price/fee to undefined
     console.error('An unknown error occurred:', error)
+    return setQuoteError({
```

Comment:
It has some duplicate code. 

Since all other branches eagerly return, you can leave a single `setQuoteError` at the end of the handling. After all, it has the same params. If you want you can leave the `console.error`

Reply:
yeah also thought this, then thought hey i dont want to do that because it works like this and I'm tired, but now I look at it again and I will change it ;)